poll-dispatch.php

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
```

Comment:
This script doesn't seem to be useful. (also any new code that calls init.php will not be accepted)

Just an option on `lnms device:poll` to control if the job is dispatched sync or async seems correct.

Reply:
I've made an async option for lnms device:poll, which resolves this